antarest/core/metrics.py

```diff
@@ -0,0 +1,92 @@
+import logging
+import os
+import time
+
+import prometheus_client
+from fastapi import FastAPI
+from prometheus_client import (
+    CollectorRegistry,
+    Counter,
+    Histogram,
+    make_asgi_app,
+)
+from prometheus_client import multiprocess
+from starlette.requests import Request
+
+from antarest.core.config import Config
+from antarest.core.exceptions import ConfigurationError
+
+logger = logging.getLogger(__name__)
+
+
+_PROMETHEUS_MULTIPROCESS_ENV_VAR = "PROMETHEUS_MULTIPROC_DIR"
```

Comment:
Add a short comment to explain the purpose of this environment variable, for instance:

```python
# The `PROMETHEUS_MULTIPROC_DIR` environment variable is used by
# the Python Prometheus client library to configure process-level metrics
# when running in a multi-process environment.
```